Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner() {
```

Comment:
you need to add a background source : https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png